modules/integration-tests/src/main/java/org/apache/fluo/integration/TestUtil.java

```diff
@@ -27,17 +27,13 @@ private TestUtil() {}
   private static final Bytes ZERO = Bytes.of("0");
 
   public static void increment(TransactionBase tx, Bytes row, Column col, int val) {
-    int prev = 0;
     String prevStr = tx.get(row, col, ZERO).toString();
-    prev = Integer.parseInt(prevStr);
-    tx.set(row, col, Bytes.of(prev + val + ""));
+    tx.set(row, col, Bytes.of(Integer.toString(Integer.parseInt(prevStr) + val)));
   }
 
   public static void increment(TransactionBase tx, String row, Column col, int val) {
-    int prev = 0;
     String prevStr = tx.gets(row, col, "0");
-    prev = Integer.parseInt(prevStr);
-    tx.set(row, col, prev + val + "");
+    tx.set(row, col, String.valueOf(Integer.parseInt(prevStr) + val));
```

Comment:
I double checked in `jshell` that `int + int + ""` does do the sum first, then convert to a String.

Also, it looks like `String.valueOf(int)` just calls `Integer.toString(int)`, so we can just use that for both.

```suggestion
    tx.set(row, col, Integer.toString(Integer.parseInt(prevStr) + val));
```